8ball/chain.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "8ball",
+  "status": "live",
+  "website": "https://8ball.info/",
+  "network_type": "mainnet",
+  "pretty_name": "8ball",
+  "chain_id": "eightball-1",
+  "bech32_prefix": "8ball",
+  "daemon_name": "8ball",
+  "node_home": "$HOME/.8ball",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uebl",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uebl"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://secp256k1.net/8ball.git",
+    "recommended_version": "v1",
+    "compatible_versions": [
+      "v1"
+    ],
+    "cosmos_sdk_version": "0.46.7",
+    "binaries": {
+      "linux/amd64": "https://8ball.info/8ball.tar.gz"
+    },
+
+    "genesis": {
+      "genesis_url": "https://8ball.info/8ball-genesis.json"
+    },
+    "versions": [
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://8ball.info/8ball.png",
```

Comment:
these URIs must point to this repo:
e.g., `https://raw.githubusercontent.com/cosmos/chain-registry/master/8ball/images/8ball.png`